src/main/java/uk/gov/hmcts/darts/ws/ContextRegistryEndpoint.java

```diff
@@ -68,4 +73,10 @@ public JAXBElement<LookupResponse> lookup(@RequestPayload JAXBElement<documentum
 
         return new ObjectFactory().createLookupResponse(lookupResponse);
     }
+
+    private static void validateRegisterBodyServiceContextIsNotNull(JAXBElement<Register> register) {
```

Comment:
Maybe we could push this into a validator?

Reply:
not sure how much worth is introducing a dedicated validator Michael for only this simple check. I would agree with you if we had plenty of different validations needed. Let's refactor to a dedicated validator if we introduced additional validations in the future